src/Player.java

```diff
@@ -202,6 +202,13 @@ public void command(String arg, Map map, Monster monster){
                         }
                         System.out.println(Arrays.asList(map.getRoomItems(map.getDungeonRoom(this.locationX, this.locationY))).toString());
                         break;
+                        case "wooden-sword":
```

Comment:
Needs proper indentation. Look at previous cases for correct format (too many tabs/spaces)